packages/prisma/index.ts

```diff
@@ -37,49 +27,32 @@ if (!isNaN(loggerLevel)) {
   }
 }
 
-// Prevents flooding with idle connections
-const prismaWithoutClientExtensions =
-  globalForPrisma.prismaWithoutClientExtensions || new PrismaClientWithoutExtension(prismaOptions);
-
-export const customPrisma = (options?: Prisma.PrismaClientOptions) =>
-  new PrismaClientWithoutExtension({ ...prismaOptions, ...options })
-    .$extends(usageTrackingExtention(prismaWithoutClientExtensions))
-    .$extends(excludeLockedUsersExtension())
-    .$extends(excludePendingPaymentsExtension())
-    .$extends(bookingIdempotencyKeyExtension())
-    .$extends(disallowUndefinedDeleteUpdateManyExtension())
-    .$extends(withAccelerate());
-
-// If any changed on middleware server restart is required
-// TODO: Migrate it to $extends
-bookingReferenceMiddleware(prismaWithoutClientExtensions);
-
-// FIXME: Due to some reason, there are types failing in certain places due to the $extends. Fix it and then enable it
-// Specifically we get errors like `Type 'string | Date | null | undefined' is not assignable to type 'Exact<string | Date | null | undefined, string | Date | null | undefined>'`
-const prismaWithClientExtensions = prismaWithoutClientExtensions
-  .$extends(usageTrackingExtention(prismaWithoutClientExtensions))
-  .$extends(excludeLockedUsersExtension())
-  .$extends(excludePendingPaymentsExtension())
-  .$extends(bookingIdempotencyKeyExtension())
-  .$extends(disallowUndefinedDeleteUpdateManyExtension())
-  .$extends(withAccelerate());
-
-export const prisma = globalForPrisma.prismaWithClientExtensions || prismaWithClientExtensions;
+export const prisma =
```

Comment:
We've removed the global prisma instances - is this desired? Won't we end up with connection pooling issues since? Dev also seem to hit max connections very quick

Reply:
as in the singleton ^